ecs-init/apparmor/apparmor.go

```diff
@@ -87,19 +101,45 @@ func LoadDefaultProfile(profileName string) error {
 		return err
 	}
 
+	includeTunablesGlobal, err := fileExists(filepath.Join(appArmorProfileDir, "tunables/global"))
+	if err != nil {
+		return err
```

Comment:
Q: If we return an error here if `tunables/global` doesn't exists, wouldn't this mean we'll never write @{PROC}=/proc/ to the profile? (please correct me if this is incorrect/the expected behavior!)

Reply:
If the file doesn't exist, `fileExists` will return `false` and no error, so we will still write the profile.
https://github.com/aws/amazon-ecs-agent/blob/ed042578fe6733eebe7cee4a60ae906414bd9427/ecs-init/apparmor/apparmor.go#L139-L141

These 3 test cases check for this (`statErrors` are the errors `os.Stat` is stubbed to return, and `profileContent` is what's expected to be written to the profile):
https://github.com/aws/amazon-ecs-agent/blob/ed042578fe6733eebe7cee4a60ae906414bd9427/ecs-init/apparmor/apparmor_test.go#L73-L97

Reply:
ah I see, thanks Tiffany!